examples/Encrypted Inference-LinearRegression.ipynb

```diff
@@ -0,0 +1,1166 @@
+{
```

Comment:
<p>MAYBE : We could mention, 2-out-of-3 sharing instead of n-1 , as here we specify it for Falcon protocol specifically.</p><p> </p><p>MAYBE : Instead of less secure,we could mention Falcon, follows a weaker adversarial model of honest majority.</p><p> </p><p>I think we cannot detect malicious behaviour  in SPDZ,as parties could manipulate the shares during the reconstruction.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/SyMPC/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Right instead I will mention, we don't yet have a malicious security type implementation of SPDZ? 

 _Instead of less secure,we could mention Falcon, follows a weaker adversarial model of honest majority._

Better. 

_MAYBE : We could mention, 2-out-of-3 sharing instead of n-1 , as here we specify it for Falcon protocol specifically._

True.